tests/ecosystem/deployment/test_deployment.py

```diff
@@ -16,7 +16,11 @@ def test_deployment():
     deploy = config.RUN['cli_params'].get('deploy')
     teardown = config.RUN['cli_params'].get('teardown')
     if not teardown or deploy:
-        ocs_install_verification()
+        if config.ENV_DATA['skip_ocs_deployment']:
+            # Verify cluster is running
+            assert is_cluster_running(config.ENV_DATA['cluster_path'])
```

Comment:
Since we are also doing this in `ocs_install_verification` would it make sense for us to check `config.ENV_DATA['skip_ocs_deployment']` after that has been done and return if true? Thus skipping the rest of the verification which seems like the goal here.

For example in `ocs_install_verification`:
```python
log.info("Verifying OCP installation")
assert is_cluster_running(config.ENV_DATA['cluster_path'])
if config.ENV_DATA['skip_ocs_deployment']:
    return
log.info("Verifying OCS installation")
...
```

Reply:
IMO even adding wording that suggests OCP verification happens in `ocs_install_verification` can lead to confusion; another way to avoid the redundancy is to remove the call to `is_cluster_running` from `ocs_install_verification`, and replace this `if/else` block with:
```
assert is_cluster_running(config.ENV_DATA['cluster_path'])
if not config.ENV_DATA['skip_ocs_deployment']:
    ocs_install_verification
```

Reply:
@zmc forgot to mention in my suggestion we could also rename to just `install_verification` but I like your approach more.